dev/com.ibm.ws.jaxrs.2.x_fat_clientProps/test-applications/cxfClientPropsApp/src/jaxrs2x/cxfClientProps/CxfClientPropsTestServlet.java

```diff
@@ -253,6 +253,37 @@ public void testChunkingThreshold(HttpServletRequest req, HttpServletResponse re
                        .post(Entity.text(sb.toString()))
                        .readEntity(String.class);
         assertEquals("30000:30000", result);
+        
+        // Repeating the tests but adding the "Expect", "100-continue" header.  In this case a 100 will
+        // be sent prior to the 200 containing the output.  The JDK will catch and handle this 100 and 
+        // JAXRS will only get the 200 response when in streaming mode (which for now is only chunking).  
+        // If not in chunking then JAXRS will receive the 100 response which will contain no returned data.
+        client = ClientBuilder.newBuilder()
+                        .property("client.ChunkingThreshold", "10000")
+                        .build();
+   
+        Response response = client.target("http://localhost:" + req.getServerPort() + "/cxfClientPropsApp/resource/chunking")
+                              .request().header("Expect", "100-continue")
+                              .post(Entity.text(sb.toString()));
+        int status = response.getStatus();
+        result = response.readEntity(String.class);
+        
+        assertEquals(200,status);
+        assertEquals("CHUNKING", result);
+
+        client = ClientBuilder.newBuilder()
+                        .property("client.ChunkingThreshold", "40000")
+                        .build();
+        response = client.target("http://localhost:" + req.getServerPort() + "/cxfClientPropsApp/resource/chunking")
+                       .request().header("Expect", "100-continue")
+                       .post(Entity.text(sb.toString()));
+        status = response.getStatus();
+        result = response.readEntity(String.class);
+
+        // If a 100 response is received then no data will be sent.
+        assertEquals(100,status);
```

Comment:
This does not seem right to me.  It is certainly inconsistent.  I don't think that we want a test case to verify that we behave incorrectly/inconsistently.  Instead, we should probably work with the CXF community to determine the right next step - possibly something like always enabling chunking when the `Expect: 100-continue` header is specified.

Reply:
I agree in principle.   However, this is our behavior now so I think I will leave it.   However I agree this should be addressed so I'll create an issue to work with CXF to resolve, indicating that this testcase will need to be adjusted when that happens.

Reply:
And it will be interesting to see if this behavior is the same for all platforms/jdks.